app/operations/beta/project_board_operations/read.rb

```diff
@@ -22,6 +24,14 @@ def call
 
       attr_reader :project_id, :current_user, :current_flow, :projects_scope
 
+      def iterations
+        @project_iterations ||= Iterations::ProjectIterations.new(project: project)
+      end
+
+      def project_iterations
+        Success(iterations)
+      end
```

Comment:
I think it's kinda confusing to have two methods with such similar names. Is it something we can unify?

```suggestion
      def iterations
        @project_iterations ||= Success(Iterations::ProjectIterations.new(project: project))
      end
```

Reply:
We can't unify due to  ` delegate :past_iterations, :current_iteration_start, to: :iterations`